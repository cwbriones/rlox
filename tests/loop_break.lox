var a = "outer";
while (true) {
    var a = "inner";
    break;
}
print "broke!"; // expect: broke!
print a; // expect: outer

while (true) {
    while (true) {
        break;
    }
    print "break 1"; // expect: break 1
    break;
}
print "break 2"; // expect: break 2
