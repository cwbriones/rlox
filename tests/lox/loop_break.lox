var a = "outer";
while (true) {
    var a = "inner";
    break;
}
print "broke!";
print a;

while (true) {
    while (true) {
        break;
    }
    print "break 1";
    break;
}
print "break 2";
