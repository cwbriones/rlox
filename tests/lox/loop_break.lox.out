broke!
outer
break 1
break 2
